chore(readme): update (#266)
aquasecurity/trivy: README.md

- Simple
  - Specify only an image name
  - See [Quick Start](#quick-start) and [Examples](#examples)
- Fast

>   - The first scan will finish within 10 seconds (depending on your network)

Reviewer: maybe add a note about consequent scans finish in single seconds

>   - The first scan will finish within 10 seconds (depending on your network)

Reviewer: ```suggestion
  - The first scan will finish within 10 seconds (depending on your network). Consequent scans will finish in single seconds.
```

>   - Most scanners take a while (~10 minutes) to fetch vulnerability information on the first run

Reviewer: comparing to clair where the server fetches the info it's true but with clair people will probably keep the server around to mitigate this first run cost. this is another benefit of trivy that it's stateless and you can just run it from scratch every time and throw it away after

>   - Most scanners take a while (~10 minutes) to fetch vulnerability information on the first run

Reviewer: ```suggestion
  - Unlike other scanners that take long to fetch vulnerability information (~10 minutes) on the first run, and encourage you to maintain the a durable vulnerability database, Trivy is stateless and requires no maintenance or preparation.
```

</details>

### Use lightweight DB

The lightweight DB doesn't contain vulnerability detail such as descriptions and references. Because of that, the size of the DB is smaller and the download is faster.

> You can search vulnerability details on the NVD website.

Reviewer: ```suggestion
To find the additional information, you can search vulnerability details on the NVD website.
```

https://nvd.nist.gov/vuln/search

> This option is useful when you don't need vulnerability details. This option should be suitable for CI/CD.

Reviewer: ```suggestion
This option is useful when you don't need vulnerability details and is suitable for CI/CD.
To find the additional information, you can search vulnerability details on the NVD website.
```

alpine:3.10 (alpine 3.10.2)
===========================
Total: 3 (UNKNOWN: 0, LOW: 1, MEDIUM: 2, HIGH: 0, CRITICAL: 0)

+---------+------------------+----------+-------------------+---------------+
| LIBRARY | VULNERABILITY ID | SEVERITY | INSTALLED VERSION | FIXED VERSION |
+---------+------------------+----------+-------------------+---------------+
| openssl | CVE-2019-1549    | MEDIUM   | 1.1.1c-r0         | 1.1.1d-r0     |
+         +------------------+          +                   +               +
|         | CVE-2019-1563    |          |                   |               |
+         +------------------+----------+                   +               +
|         | CVE-2019-1547    | LOW      |                   |               |
+---------+------------------+----------+-------------------+---------------+
```
</details>

### Deprecated options

`--only-update`, `--refresh` and `--auto-refresh` are deprecated since they are unnecessary now. These options will be removed at the next version

# Continuous Integration (CI)

Scan your image built in Travis CI/CircleCI. The test will fail if a vulnerability is found. When you don't want to fail the test, specify `--exit-code 0` .

> **Note**: The scan is already fast enough so you don't require caching now. Most scans will finish within a few seconds (depending on image size)

Reviewer: ```suggestion
Since in automated scenarios such as CI/CD you only interested in the end result, and not the full report, use the `--light` flag to optimize  for this scenario and get fast results.
```

And as many people know, it is difficult to select a `Clair` client because many clients are deprecated.

> Trivy is a stand-alone tool and can scan very fast. This means it's very easy to use in CI/CD.

Reviewer: see the comment of statelessness from the features section
